packages/contracts/src/interfaces/IPaymaster.sol

```diff
@@ -40,7 +26,33 @@ interface IPaymaster is IERC165 {
     }
 
     /**
-     * Return the Gas Limits and msg.data max size constants used by the Paymaster.
+     * @notice Return the Gas Limits for Paymaster's functions and maximum msg.data length values for this Paymaster.
```

Comment:
There is a problem with deep documentation in the source: you're putting on-chain security assumptions that belong to our doc.